src/fillTank.test.js

```diff
@@ -1,11 +1,104 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('should be declared', () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('If amount is empty, then full tank is ordered', () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 50);
```

Comment:
Ensure that the `fillTank` function correctly handles the case where the amount is not provided, and fills the tank to its maximum capacity. Verify that the customer's money is deducted correctly based on the full tank capacity and fuel price.